src/Components/Common/Drawer/Drawer.tsx

```diff
@@ -61,6 +62,49 @@ export default function MiniDrawer(props: any): ReactElement {
     //window.location.reload();
   }
 
+  function loadAdmin():any {
+    if (role=="ADMIN")
+    {
```

Comment:
Use enums with the two values as ROLE_ADMIN and ROLE_CLIENT, instead of using "ADMIN" as a String